targets/TARGET_NUVOTON/TARGET_M480/mbed_lib.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@0xc0170 There is incompatibility issue with some sample code. I will move to `targets.json` later.